bootstrap.py

```diff
@@ -765,6 +772,8 @@ class MEOptions:
         options.skip.append('puppet')
     if not options.removepkgs:
         options.skip.append('remove-obsolete-packages')
+    if options.new_capsule:
```

Comment:
Does it show really badly that I don't do this for a living ;) Will change, new pull request to follow